app/code/community/Zip/Payment/controllers/CheckoutController.php

```diff
@@ -63,7 +96,14 @@ public function startAction()
             $checkoutId = $this->getHelper()->getCheckoutIdFromSession();
             $redirectUrl = $this->getHelper()->getCheckoutRedirectUrlFromSession();
         }
-
+        $mode = $this->getHelper()->getConfig()->getValue(Zip_Payment_Model_Config::CONFIG_CHECKOUT_DISPLAY_MODE_PATH);
+        if ($mode == 'lightbox') {
+            $checkoutSession = Mage::getSingleton('checkout/session');
+            $quote = $checkoutSession->getQuote();
+            if ($quote && strtoupper($quote->getQuoteCurrencyCode()) === 'NZD') {
```

Comment:
@benzhangzip can we use \Zip\Model\CurrencyUtil::CURRENCY_NZD

Reply:
Fixed